Framework/CurveFitting/src/Algorithms/EstimateFitParameters.cpp

```diff
@@ -102,6 +106,39 @@ void fixBadParameters(CostFunctions::CostFuncFitting &costFunction,
   }
 }
 
+/// A class for sorting and storing sets of best function parameters.
+class BestParameters {
+  /// Maximum size of the store
+  size_t m_size;
+  /// Actual storage.
+  std::map<double, GSLVector> m_params;
```

Comment:
Is the order important here? If all we want is a lookup then `unordered_map` should be preferred: http://stackoverflow.com/questions/2196995/is-there-any-advantage-of-using-map-over-unordered-map-in-case-of-trivial-keys